### ISSUE TITLE
DASH/ClearKey: Skipping atom with length > 2147483647 (unsupported)

### ISSUE THREAD

**Issue by adamhewitt627** [google/ExoPlayer#4137]
### Issue description
When playing a ClearKey DASH file, throws `com.google.android.exoplayer2.ParserException: Skipping atom with length > 2147483647 (unsupported).` The same URL will play in Edge, Safari, Chrome and Firefox at http://ampdemo.azureedge.net/azuremediaplayer.html

### Reproduction steps
1. Add sample playlist data to `media.exolist.json`
2. Optionally hard-code a local DRM callback. (The presence of it doesn't seem to make a difference, but may if there is a partial bug that is addressed which allows it to get far enough to request DRM.)
```Java
new LocalMediaDrmCallback("{\"keys\":[{\"kid\":\"AVxCgq4LrkGh2e+FbAMnCA\",\"k\":\"qyCmHP0AIq5d63qblMFl8g\",\"type\":\"temporary\",\"kty\":\"oct\"}]}".getBytes())
```

### Link to test content
See email with test content. 4 streams are referenced, the one marked `ERROR` reproduces while the others will play.

### Version of ExoPlayer being used
Checkout of sample application at 4b531dcc5

### Device(s) and version(s) of Android being used
- HTC Nexus 9, Android 7.1.1
- Samsung Nexus 10, Android 5.1.1
- Pixel C, Android 8.0.0

**Comment by ojw28** [comment]
It seems this is not really ClearKey content, at least not according to our definition of what ClearKey content is. I found this related thread (for a different player). I think the same answer basically holds true here as well:

https://github.com/google/shaka-player/issues/358#issuecomment-217452882

We do not support the type of encryption Microsoft are using for these streams. Closing as unsupported.
